src/main/java/racingcar/service/Racing.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.service;
+
+import java.util.List;
+import racingcar.exception.RacingException;
+import racingcar.model.Car;
+import racingcar.strategy.ModeType;
+import racingcar.util.CarNameSeparator;
+import racingcar.util.CarsCreator;
+import racingcar.util.RacingResultCalculator;
+
+public class Racing {
+    private final List<Car> racingCars;
+
+    public Racing(String carNames) {
+        this.racingCars = createCar(validate(carNames));
+    }
+
+    private List<String> validate(String carNames) {
+        RacingException.separatorException(carNames);
+        List<String> carNameList = CarNameSeparator.splitCarNameWithoutSpace(carNames);
+        RacingException.emptyException(carNameList);
+        RacingException.duplicationException(carNameList);
+        return carNameList;
+    }
+
+    private List<Car> createCar(List<String> carNamesList) {
+        return CarsCreator.createCarNameToCar(carNamesList);
+    }
+
+    public void tryMovingCars() {
+        racingCars.forEach(Car::tryMoving);
+    }
+
+    public void setRacingCarMode(ModeType modeType) {
+        racingCars.forEach(car -> car.setMode(ModeType.createCarMode(modeType)));
+    }
+
+    public List<String> getRacingWinner() {
+        Integer racingWinnerRecord = RacingResultCalculator.calculateCarRacingWinnerRecord(racingCars);
+        return RacingResultCalculator.calculateCarRacingWinner(racingCars, racingWinnerRecord);
+    }
+
+    public List<Car> getRacingCars() {
+        return racingCars;
```

Comment:
```suggestion
        return new ArrayList<>(racingCars);
```
이렇게 바꾸면 외부 참조에 대해 방어할 수 있을 것 같습니다!

Reply:
`Collections.unmodifiableList(racingCars)` 해당 메서드를 사용해도 외부 참조에 대해 방어가 가능하다고 하는데, 어떤 방식을 사용해야 할까요?